### PR TITLE
Crash during mounting

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Background, with logfile - Panic message printed in Logfile
Background, without logfile - Panic message printed in Syslog
Foreground, with logfile - Panic message printed in Logfile
Foreground, without logfile - Panic message printed on Console

### Testing
1. Manual - Tested manually by putting panic message before mounting
2. UnitTest - N/A
3. IntegrationTest - N/A

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -422,7 +422,18 @@ func run() (err error) {
 	return
 }
 
+func handlePanic() {
+	// detect if panic occurred or not
+	a := recover()
+	if a != nil {
+		logger.Info("Panic: ", a)
+		os.Exit(1)
```
Comment (raj-prince) [inline main.go:430]: What happens if we don't keep os.Exit(1)?

Comment (Tulsishah) [reply main.go]: Nothing, but I thought exit status 1 would show an error, as I found it in many places. It Fixed.

File: main.go [hunk 2]
```diff
@@ -422,7 +422,17 @@ func run() (err error) {
 	return
 }
 
+func handlePanic() {
+	// detect if panic occurred or not
```
Comment (raj-prince) [inline main.go:426]: Please fix the comment - "Detect if panic happens in main go routine."

**Review by raj-prince — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (Tulsishah) [reply main.go]: Fixed

**Pull request closed without merging**